firebase-database/src/main/java/com/google/firebase/database/core/Repo.java

```diff
@@ -574,12 +574,19 @@ public void onCancelled(@NonNull DatabaseError error) {
                             new ValueEventRegistration(repo, listener, spec);
```

Comment:
Can we use a KeepSyncedEventRegistration for the get query?

https://github.com/firebase/firebase-android-sdk/blob/884e53770ef0fb224b9401ade32a4f8fc4bc4e35/firebase-database/src/main/java/com/google/firebase/database/core/SyncTree.java#L799-L848